helm/charts/kratos/templates/job-migration.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if and  ( .Values.kratos.automigration.enabled ) ( eq .Values.kratos.automigration.type "job" ) }}
+{{- include "kratos.automigration.typeVerification" . -}}
```

Comment:
hmm, I think type verification should be first cause in the first `if` you are already checking if value is "job"